### PR TITLE
Fix the cleanup

### PR DESCRIPTION
**Pull request by TechnichalDoge**
This fixes the cleanup command not working on the hoverballs, also disables its collision setting.

### CODE REVIEW

File: lua/weapons/gmod_tool/stools/hoverball_spawner.lua [hunk 1]
```diff
@@ -27,6 +29,8 @@ function TOOL:LeftClick( trace )
         if (IsValid(trace.Entity)) then
             local weld = constraint.Weld( entity, trace.Entity, 0, trace.PhysicsBone, 0, true , false )
         end
+            self:GetOwner():AddCount( "Marums Hovers", entity )
```
Comment (plally) [inline lua/weapons/gmod_tool/stools/hoverball_spawner.lua]: why is this here? this function is used for sbox entity limits not cleanup functions

Comment (TechnichalDoge) [reply lua/weapons/gmod_tool/stools/hoverball_spawner.lua]: /shrug
Ech i was looking at the original gmod hoverball code, and i just basically replicated it.

**Comment by TechnichalDoge** [comment]
Im just gonna close this and redo it.
